### PR TITLE
Build error: Pin @types/passport package to 0.4.0

### PR DESCRIPTION
**Pull request by anoadragon453**
[@types/passport](https://www.npmjs.com/package/@types/passport) updated and made a backward-incompatible type change which causes the following error when attempting to run `npm run start:web`:

```

ERROR in [at-loader] ./src/api/security/MatrixSecurity.ts:44:49 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/security/MatrixSecurity.ts:79:107 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/security/MatrixSecurity.ts:87:100 
    TS2345: Argument of type 'User' is not assignable to parameter of type 'ILoggedInUser'.
  Type 'User' is missing the following properties from type 'ILoggedInUser': userId, token

ERROR in [at-loader] ./src/api/admin/AdminAppserviceService.ts:53:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminCustomSimpleBotService.ts:57:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminCustomSimpleBotService.ts:68:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminCustomSimpleBotService.ts:79:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminGitterService.ts:67:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminGitterService.ts:85:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminGitterService.ts:104:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionIntegrationsService.ts:93:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionIntegrationsService.ts:119:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionIntegrationsService.ts:133:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIntegrationsService.ts:33:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIntegrationsService.ts:48:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIntegrationsService.ts:63:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIrcService.ts:45:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIrcService.ts:73:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIrcService.ts:101:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIrcService.ts:127:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminIrcService.ts:147:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminNebService.ts:58:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminNebService.ts:72:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminNebService.ts:93:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminNebService.ts:112:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminSlackService.ts:67:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminSlackService.ts:85:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminSlackService.ts:104:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionStickerService.ts:104:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionStickerService.ts:131:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminStickerService.ts:56:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminTelegramService.ts:78:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminTelegramService.ts:107:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminUpstreamService.ts:49:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminWebhooksService.ts:70:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/admin/AdminWebhooksService.ts:97:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionGitterService.ts:24:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionGitterService.ts:38:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionGitterService.ts:53:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionIrcService.ts:25:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionIrcService.ts:41:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionIrcService.ts:57:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionSlackService.ts:26:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionSlackService.ts:41:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionSlackService.ts:56:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionSlackService.ts:74:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionSlackService.ts:86:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionSlackService.ts:101:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionTelegramService.ts:31:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionTelegramService.ts:55:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionTelegramService.ts:78:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionWebhooksService.ts:60:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionWebhooksService.ts:69:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/dimension/DimensionWebhooksService.ts:78:50 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/matrix/MatrixAccountService.ts:31:15 
    TS2322: Type 'User' is not assignable to type 'ILoggedInUser'.

ERROR in [at-loader] ./src/api/matrix/MatrixAccountService.ts:39:45 
    TS2345: Argument of type 'User' is not assignable to parameter of type 'ILoggedInUser'.

ERROR in [at-loader] ./src/api/matrix/MatrixTermsService.ts:33:54 
    TS2345: Argument of type 'User' is not assignable to parameter of type 'ILoggedInUser'.

ERROR in [at-loader] ./src/api/scalar/ScalarService.ts:48:52 
    TS2339: Property 'userId' does not exist on type 'User'.

ERROR in [at-loader] ./src/api/scalar/ScalarService.ts:61:54 
    TS2345: Argument of type 'User' is not assignable to parameter of type 'ILoggedInUser'.
```

TL;DR they changed the type of `User.userId` from `any` to `User`.

Pin the version of the package to fix new clones until we update the package to the latest version.

### CODE REVIEW

**Comment by turt2live** [comment]
This doesn't seem to be an issue anymore. If it is, please open a new PR.

Apologies for the delay in review :(
